metricflow/sql/render/expr_renderer.py

```diff
@@ -255,3 +256,13 @@ def visit_ratio_computation_expr(self, node: SqlRatioComputationExpression) -> S
             sql=f"{numerator_sql} / {denominator_sql}",
             execution_parameters=execution_parameters,
         )
+
+    def visit_between_expr(self, node: SqlBetweenExpression) -> SqlExpressionRenderResult:  # noqa: D
+        rendered_column_arg = self.render_sql_expr(node.column_arg)
+        rendered_left_value = self.render_sql_expr(node.left_value)
+        rendered_right_value = self.render_sql_expr(node.right_value)
+
+        return SqlExpressionRenderResult(
+            sql=f"{rendered_column_arg.sql} BETWEEN {rendered_left_value.sql} AND {rendered_right_value.sql}",
+            execution_parameters=SqlBindParameters(),
```

Comment:
You'll want to combine the bind parameters from the arguments and put them here.

Reply:
added execution parameters